.github/workflows/publish.yml

```diff
@@ -0,0 +1,28 @@
+name: Publish
+on:
+  push:
+    branches:
+      - "master"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
this is an outdated recommendation from the Changesets action - we now automatically "deepen" the git tree so it shouldn't be required to fetch all the git history by default
```suggestion
```

I probably should remove that from the `changesets/action`'s README :sweat_smile: